Utils/Dataflow/019_esFormat/run.sh

```diff
@@ -1,23 +1,32 @@
-#/usr//bin/env bash
+#/usr/bin/env bash
 
 base_dir=$(cd "$(dirname "$(readlink -f "$0")")"; pwd)
 
+ES_CONFIG=$base_dir/../../Elasticsearch/config/es
+CONFIG_DEFAULT=TRUE
+
 usage() {
-  echo "USAGE:
-$(basename "$0") [-c CONFIG] [--] [ARGS]
+  echo "usage: $(basename "$0") [-h] [-c CONFIG] [--] [ARGS]
   
-PARAMETERS:
-  CONFIG -- configuration file
-  ARGS   -- arguments to be passed to the PHP script
-"
-}
+optional arguments:
+  -h, --help   show this help message and exit
 
-ES_CONFIG=$base_dir/../../Elasticsearch/config/es
+  -c CONFIG    configuration file with parameters for
+               accessing data in Elasticsearch
+               Default value: $ES_CONFIG
```

Comment:
"Accessing data in ES" sounds a bit confusing, as if the stage was accessing ES -- while it does not.
Can you try to reword it a bit?
For me, the most accurate description would be "config.file with parameters required to prepare data for indexing in ES" (and the most "as-is" -- "config.file containing ES_INDEX definition").

Reply:
Done, edc034c.